src/Application.vala

```diff
@@ -90,6 +90,8 @@ public class Sequeler.Application : Gtk.Application {
 
     protected override void activate () {
         var window = new Sequeler.Window (this);
+        Gtk.Settings.get_default().set_property("gtk-theme-name", "elementary");
```

Comment:
Why those are enforced here?

Reply:
This might causes issues if the user doesn't have the elementary gtk/icons theme installed.

Reply:
You're right, but without it, `office-database` icon (on ConnectionDialog) is not resolved.

Not sure how to deal with sequeler build outside elementary theme